guildwars2/guild/general.py

```diff
@@ -55,27 +64,35 @@ class GeneralGuild:
         except discord.Forbidden:
             await ctx.send("Need permission to embed links")
 
-    @guild.command(name="members")
+    @guild.command(name="members", usage="<guild name>")
     @commands.cooldown(1, 20, BucketType.user)
-    async def guild_members(self, ctx, *, guild_name: str):
+    async def guild_members(self, ctx, *, guild_name=None):
         """Get list of all members and their ranks.
```

Comment:
Unrelated, but this help message ends up a bit misleading as it doesnt display all members. Longer version of help message explains it, but the short version (first line) doesn't